pkg/attestation/crafter/materials/materials.go

```diff
@@ -57,6 +57,10 @@ func uploadAndCraft(ctx context.Context, input *schemaapi.CraftingSchema_Materia
 	}
 	defer result.r.Close()
 
+	if result.size == 0 {
```

Comment:
this is not strictly part of this error but it's an additional protection to let the user know.